src/VisualStudio/Core/Def/Implementation/ProjectSystem/RuleSets/VisualStudioRuleSetManager.cs

```diff
@@ -50,5 +50,10 @@ public void Dispose()
 
             _ruleSetFileMap.Clear();
         }
+
+        void IDisposable.Dispose()
```

Comment:
Tests still treat it as a disposable unit, and that seems appropriate as a simple cache.